zeebe/expression-language/src/main/java/io/camunda/zeebe/el/impl/FeelVariableContext.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.el.impl;
+
+import io.camunda.zeebe.el.EvaluationContext;
+import org.camunda.feel.context.CustomContext;
+import org.camunda.feel.context.VariableProvider;
+import scala.Option;
+import scala.collection.Iterable;
+import scala.collection.immutable.List$;
+
+final class FeelVariableContext extends CustomContext {
```

Comment:
❓ Similar to the `FeelEvaluationResult`, should this class be `public`?

Reply:
Same as above, as it's an implementation detail I'd say no.